css-four-column-layout/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>CSS four column layout</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS four column layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@100&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@700&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="container">
+  <div class="row">
+    <div class="column-full">
+      <h2>How It Works</h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="column-full">
+      <div class="image-storage">
+        <img class="images" src="images/screen-1.png" alt="screen-4">
+        <img class="images" src="images/screen-2.png" alt="screen-4">
+        <img class="images" src="images/screen-3.png" alt="screen-4">
+        <img class="images" src="images/screen-4.png" alt="screen-4">
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="column-full">
+      <div class="image-storage">
+        <img class="icons" src="images/switch-icon.png" alt="switch">
+        <img class="icons" src="images/filter-icon.png" alt="filter">
+        <img class="icons" src="images/slider-icon.png" alt="slider">
+        <img class="icons" src="images/bars-icon.png" alt="bars">
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="column-full">
+      <div class="image-storage">
+        <h3>App Switching</h3>
+        <h3>Trip Filtering</h3>
+        <h3>Service Filters</h3>
+        <h3>Stats + Mileage</h3>
+      </div>
+    </div>
+  </div>
+  <footer>
+    <div class="row">
+      <div class="column-full">
+        <div class="image-storage">
+          <p>Toggles all your apps on and off.</p>
+          <p>Evaluates your trip requests</p>
+          <p>Accepts only your ideal trips</p>
+          <p>Tracks your trips across all apps</p>
+        </div>
+      </div>
+    </div>
+  </footer>
```

Comment:
I can see why you broke these pieces of the layout into rows based on how they look in terms of "layout", but this makes the content of each column seem separate from one another. You should have one `row` with four `columns`, each of which holds the image, icon, and text for that column